YIF_Backend/Controllers/SuperAdminController.cs

```diff
@@ -329,5 +329,23 @@ public async Task<IActionResult> GetIoEAdminIdByIoEId(string ioEId)
             var result = await _superAdminService.GetIoEAdminIdByIoEId(ioEId);
             return Ok(result.Object);
         }
+
+        /// <summary>
+        /// Delete Specialty (sets its asp.net user IsDeleted to true).
+        /// </summary>
+        /// <returns>Success message</returns>
+        /// <response code="200">Success message</response>
+        /// <response code="404">Not found message</response>
```

Comment:
add code 403 pls cuz you it's authorized endpoint

Reply:
Ok. Done)